test/project_types/rails/commands/create_test.rb

```diff
@@ -45,7 +45,7 @@ def test_can_create_new_app
         expect_command(%W(#{gem_path}/bin/rails new --skip-spring --database=sqlite3 test-app))
         expect_command(%W(#{gem_path}/bin/bundle install),
                        chdir: File.join(@context.root, 'test-app'))
-        expect_command(%W(#{gem_path}/bin/spring stop),
+        expect_command(["spring stop || true"],
```

Comment:
(I don't fully understand why the previous test included the `gem_path`).

Reply:
If I remember correctly, it has to do with Windows.  GEM_HOME is usually set in the environment for mac (especially our environments) but on Windows, one has to figure it out (by retrieving output from `gem environment home`).

Hence `Gem.binary_path_for()` (used by `syscall()` in `rails/commands/create.rb`) will default to just the binary name on mac but on Windows will be a fullpath name, picking up GEM_HOME as derived from `gem environment home`.